source/simulator/solver.cc

```diff
@@ -791,6 +791,14 @@ namespace aspect
     LinearAlgebra::BlockVector distributed_stokes_solution (introspection.index_sets.stokes_partitioning,
                                                             mpi_communicator);
 
+    // We will need the Stokes block indices a lot below, shorten their names
+    const unsigned int velocity_block_index = introspection.block_indices.velocities;
+    const unsigned int pressure_block_index = (parameters.include_melt_transport) ?
+                                              introspection.variable("fluid pressure").block_index
+                                              : introspection.block_indices.pressure;
+    (void) velocity_block_index;
```

Comment:
I know it already was like this, but why are we casting to void here?

Reply:
We are casting to void to silence compiler warnings. In some branch of the code the indices may only be used in asserts, and if those are removed in release mode then the variables may not be used. The compiler warns about this, but if we cast to void the compiler accepts this as a proper use of the variable.